engines/pytorch/pytorch-engine/src/test/java/ai/djl/pytorch/jni/IValueUtilsTest.java

```diff
@@ -42,4 +43,54 @@ public void getInputsTestTupleSyntax() {
             Arrays.stream(iValues).forEach(IValue::close);
         }
     }
+
+    @Test
+    public void testMapOfTensor() {
+        try (NDManager manager = NDManager.newBaseManager()) {
+            NDArray array1 = manager.zeros(new Shape(1));
+            array1.setName("input1.key1");
+            NDArray array2 = manager.ones(new Shape(1));
+            array2.setName("input1.key2");
+            NDArray array3 = manager.zeros(new Shape(1));
+            array3.setName("input2.key1");
+            NDArray array4 = manager.ones(new Shape(1));
+            array4.setName("input2.key2");
+            NDArray array5 = manager.ones(new Shape(1));
+            array5.setName("input2.key3");
+            NDList input = new NDList(array1, array2, array3, array4, array5);
+            // the NDList is mapped to (input1: Dict(str, Tensor), input2: Dict(str, Tensor))
```

Comment:
Any way to generate some NDList like 

> (input1: Dict(str, **list(Tensor)**), input2: Dict(str, **list(Tensor)**))

Reply:
1. We don't support this currently, it's possible to add nested IValue support.
2. But you have to use IValue directly, and use PyTorch specific API, which doesn't work with Translator pattern. something like: https://github.com/deepjavalibrary/djl/blob/master/engines/pytorch/pytorch-engine/src/test/java/ai/djl/pytorch/integration/IValueTest.java#L184-L191

Reply:
I created a PR for your issue: https://github.com/deepjavalibrary/djl/pull/1765